maps/~mapsystem/maps.dm

```diff
@@ -269,8 +269,6 @@ var/global/const/MAP_HAS_RANK = 2		//Rank system, also togglable
 		var/found_whitelisted_job = FALSE
 		for(var/datum/job/job as anything in SSjobs.primary_job_datums)
 			if((species.type in job_to_species_whitelist[job.type]) || (job.type in species_to_job_whitelist[species.type]))
-				LAZYDISTINCTADD(species_to_job_whitelist[species.type], job.type)
```

Comment:
i thiiink if `job_to_species_whitelist[job.type]` is set then `species_to_job_whitelist[species.type]` should also be set, but not vice versa? although i guess it should specifically only set it if the list already exists otherwise it will implicitly blacklist them from all other jobs... oh wait no we discussed this and it's fine